### PR TITLE
add: `docker-bin`

### PR DESCRIPTION
**Pull request by oklopfer**

### CODE REVIEW

File: packages/docker-bin/docker-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name="docker-bin"
+gives="docker"
+replace="${gives}"
+repology=("project: ${gives}")
+version="23.0.1"
+depends="procps git iptables"
+description="Docker Engine Static Binaries"
+
+if [ $(dpkg --print-architecture) = arm64 ]; then
```
Comment (Henryws) [inline packages/docker-bin/docker-bin.pacscript:9]:
```suggestion
if [[ $CARCH == "amd64" ]]; then
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/docker-bin/docker-bin.pacscript (hunk 1 above)
Comment (oklopfer) [reply packages/docker-bin/docker-bin.pacscript:9]: Do you mean $CARCH == arm64 or $CARCH !== amd64

Comment (Henryws) [reply packages/docker-bin/docker-bin.pacscript:9]: If `$CARCH == 'amd64'`

Comment (oklopfer) [reply packages/docker-bin/docker-bin.pacscript:9]: why tho

**Review by Henryws — APPROVED** [review]

**Pull request merged**